From b1d62f15c5a4b89df12aca33a58fe9405bf53a67 Mon Sep 17 00:00:00 2001
From: Michael Forney <mforney@mforney.org>
Date: Tue, 19 Nov 2019 19:33:01 -0800
Subject: [PATCH] libctf: Remove usage of __thread

We only use libctf for building single-threaded binutils, so __thread
is not necessary.
---
 libctf/ctf-archive.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libctf/ctf-archive.c b/libctf/ctf-archive.c
index 73d772ffc3..4896aa721b 100644
--- a/libctf/ctf-archive.c
+++ b/libctf/ctf-archive.c
@@ -45,7 +45,7 @@ static int arc_mmap_writeout (int fd, void *header, size_t headersz,
 static int arc_mmap_unmap (void *header, size_t headersz, const char **errmsg);
 
 /* bsearch() internal state.  */
-static __thread char *search_nametbl;
+static char *search_nametbl;
 
 /* Write out a CTF archive to the start of the file referenced by the passed-in
    fd.  The entries in CTF_FILES are referenced by name: the names are passed in
-- 
2.25.1

